### DISCUSSION TITLE
PrivateGPT Segmentation Fault (core dumped) Error

### DISCUSSION THREAD

**Discussion by BennisonDevadoss** [imartinez/privateGPT#1145] [Q&A]
I've been using PrivateGPT, and it suddenly crashed with a "Segmentation fault (core dumped)" error, which stopped the process.

exact exception is as follows
```none
make: *** [Makefile:36: run] Segmentation fault (core dumped)
```

I investigated, and it appears that the exception might be connected to the computer's memory (RAM), although I'm not entirely certain. PrivateGPT typically uses about 5.5GB of memory. When I checked the system using the top command, I noticed it was using more than 5GB of memory. For a clearer picture, please see the snapshot below.

![image](https://github.com/imartinez/privateGPT/assets/89965371/81c470da-5f3e-4b7f-aa4a-1000c91c27ca)

**System Configuration:**
- **Operating System:** Ubuntu 22.04.3 LTS
- **CPU:** Intel(R) Xeon(R) CPU @ 2.00GHz x 8
- **GPU:** NVIDIA Corporation TU104GL [Tesla T4] 
- **CUDA:** 12.3
- **RAM:** 30GB
- **Python Version:** Python 3.11.6

**Additional Info about GPU:**
![image](https://github.com/imartinez/privateGPT/assets/89965371/f6ce530e-5592-45d5-9ff5-047c9e01b98b)

I would greatly appreciate any assistance in diagnosing and resolving this error.

**Comment by touqeerShah** [comment]
Try run this `PYTORCH_MPS_HIGH_WATERMARK_RATIO=0.0 ./webui.sh --no-half` then run module again maybe solve issue

**Reply by BennisonDevadoss** [reply]
Can you explain more about this command, What this command does in behind?
I would like to know it before use it.
